external/shellclient/v1/messages.go

```diff
@@ -0,0 +1,27 @@
+package v1
+
+const (
```

Comment:
Maybe put this into a subpackage called `pkg/docker_exec_ws/const` since, there is no reason they should be exposed (public) from the `pkg/docker_exec_ws` package, right?

But I agree, having them public and documented is nice... so a subpackage might be a good way to hide them from people using the package to reduce confusion...